src/components/queryBuilder/utils.ts

```diff
@@ -21,14 +21,15 @@ import {
 export const isBooleanType = (type: string): boolean => {
   return ['boolean'].includes(type.toLowerCase());
 };
+
 export const isNumberType = (type: string): boolean => {
-  const numericTypes = ['int', 'float', 'decimal'];
-  const match = numericTypes.find((t) => type.toLowerCase().includes(t));
-  return match !== undefined;
+  return ['int', 'float', 'decimal'].some((t) => type.toLowerCase().includes(t));
 };
+
 export const isDateType = (type: string): boolean => {
-  return ['date', 'datetime'].includes(type.toLowerCase());
+  return type.toLowerCase().includes('date');
```

Comment:
Maybe we should use `startsWith` instead to exclude matches against `Array(Date)`, tuples `Tuple(UInt8, Date)`?

Reply:
Changing it to `startsWith` instead of `includes` sounds like a good idea.